distuv/exponential.go

```diff
@@ -199,13 +200,13 @@ func (e Exponential) StdDev() float64 {
 // of the samples.
 //
 // If weights is nil, the weights are assumed to be 1, otherwise panics if
-// len(samples) != len(weights). Panics if len(suffStat) != 1.
+// len(samples) != len(weights). Panics if len(suffStat) != NumSuffStat().
 func (Exponential) SuffStat(samples, weights, suffStat []float64) (nSamples float64) {
```

Comment:
A side note in case we want to stabilize the API one day: the destination slice is usually the first.